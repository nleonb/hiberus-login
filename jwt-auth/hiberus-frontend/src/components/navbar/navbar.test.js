import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from "react-router-dom";

import Navbar from './navbar';

describe('Test Navbar component ', () => {
    it('renders correctly', () => {
        const component = renderer.create(
            <MemoryRouter initialentries="{['/']}">
                <Navbar />
            </MemoryRouter>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});