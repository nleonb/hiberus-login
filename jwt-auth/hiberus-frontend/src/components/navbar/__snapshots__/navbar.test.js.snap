// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar component  renders correctly 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-light"
  role="navigation"
>
  <div
    className="container"
  >
    <div
      className="navbar-menu"
      id="navbarBasicExample"
    >
      <div
        className="navbar-start"
      >
        <a
          className="navbar-item"
          href="/dashboard"
        >
          Home
        </a>
      </div>
      <div
        className="navbar-end"
      >
        <div
          className="navbar-item"
        >
          <div
            className="buttons"
          >
            <button
              className="button is-dark"
              onClick={[Function]}
            >
              Log Out
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</nav>
`;
